doc/cordova.txt

```diff
@@ -1,14 +1,11 @@
 Synopsis
-
```

Comment:
Maybe somewhere here, we should mention that the user can get more info about command 'x' by running: `cordova x --help`